src/main/java/gregtech/common/gui/widget/orefilter/OreFilterTestSlot.java

```diff
@@ -1,95 +1,88 @@
 package gregtech.common.gui.widget.orefilter;
 
-import gregtech.api.gui.GuiTextures;
-import gregtech.api.gui.IRenderContext;
-import gregtech.api.gui.resources.TextureArea;
-import gregtech.api.gui.widgets.ImageWidget;
-import gregtech.api.gui.widgets.WidgetGroup;
-import gregtech.api.util.LocalizationUtils;
-import gregtech.api.util.Position;
+import gregtech.api.unification.OreDictUnifier;
 import gregtech.api.util.function.BooleanConsumer;
 import gregtech.api.util.oreglob.OreGlob;
+import gregtech.common.covers.filter.oreglob.impl.ImpossibleOreGlob;
 
-import net.minecraft.client.renderer.GlStateManager;
-import net.minecraft.client.resources.I18n;
-import net.minecraft.item.ItemStack;
-import net.minecraftforge.fml.relauncher.Side;
-import net.minecraftforge.fml.relauncher.SideOnly;
-
+import com.cleanroommc.modularui.api.drawable.IKey;
+import com.cleanroommc.modularui.widgets.ItemSlot;
 import it.unimi.dsi.fastutil.objects.Object2BooleanAVLTreeMap;
 import it.unimi.dsi.fastutil.objects.Object2BooleanMap;
 import it.unimi.dsi.fastutil.objects.Object2BooleanMaps;
 import org.jetbrains.annotations.Nullable;
 
-import java.util.Arrays;
 import java.util.Collections;
 import java.util.Set;
+import java.util.function.Supplier;
 import java.util.stream.Collectors;
 
 /**
  * @author brachy84
  */
-public abstract class OreFilterTestSlot extends WidgetGroup {
-
-    private final ImageWidget match;
-    private final ImageWidget noMatch;
+public class OreFilterTestSlot extends ItemSlot {
 
-    @Nullable
-    private OreGlob glob;
+    private final ItemOreFilterTestSlot slot;
+    private Supplier<OreGlob> globSupplier = ImpossibleOreGlob::getInstance;
     private boolean expectedResult = true;
-
-    @Nullable
-    private TextureArea slotIcon = GuiTextures.SLOT;
-
     @Nullable
     private BooleanConsumer onMatchChange;
-
     private Object2BooleanMap<String> testResult;
     private MatchType matchType = MatchType.INVALID;
     private boolean matchSuccess;
 
-    private boolean initialized = false;
+    // private boolean initialized = false;
```

Comment:
What is the status on this class's commented out code?

Reply:
this field is no longer needed